packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/OutputTypes/FieldDeclaration.cs

```diff
@@ -6,27 +6,31 @@
 
 namespace Microsoft.Generator.CSharp
 {
-    public sealed record FieldDeclaration(FormattableString? Description, FieldModifiers Modifiers, CSharpType Type, string Name, ValueExpression? InitializationValue)
+    public sealed class FieldDeclaration
     {
+        public FormattableString? Description { get; }
+        public FieldModifiers Modifiers { get; }
+        public CSharpType Type { get; }
+        public string Name { get; }
+        public ValueExpression? InitializationValue { get; }
         public string Accessibility => (Modifiers & FieldModifiers.Public) > 0 ? "public" : "internal";
```

Comment:
lets remove this property as it was the old way to track FieldModifiers and is now a duplicate